modules/apps/remote-app/remote-app-service/src/main/java/com/liferay/remote/app/internal/upgrade/RemoteAppServiceUpgrade.java

```diff
@@ -40,6 +40,11 @@ public void register(Registry registry) {
 			"2.0.0", "2.1.0",
 			new com.liferay.remote.app.internal.upgrade.v2_1_0.
 				ResourcePermissionsUpgradeProcess());
+
```

Comment:
This will conflict with https://github.com/brianchandotcom/liferay-portal/pull/109527/files#diff-a28f3dc8330cd0f70dd385be609e129a2e5899b99a7b85da17144909b23d342dR44 :tada:

Reply:
😞 I want it to send this today as I'm three days over the weekly deadline...

Reply:
https://media.giphy.com/media/7FgDCnVUXgEJJiT68U/giphy-downsized-large.gif

Reply:
You won this time @izaera ... Next time won't be so easy...

Reply:
![no comments](https://media.giphy.com/media/8gQR12M5d4kPMRYoC1/giphy.gif)

Reply:
LOL 😂